app/controllers/admin/subscriptions_controller.rb

```diff
@@ -25,7 +26,9 @@ def index
             render :setup_explanation
           end
         end
-        format.json { render_as_json @collection, ams_prefix: params[:ams_prefix], payment_method_tags: payment_method_tags_by_id }
+        format.json {
+          render_as_json @collection, ams_prefix: params[:ams_prefix],
+          payment_method_tags: payment_method_tags_by_id }
```

Comment:
This is definitely an improvement, but it would be good to have the closing brace on a new line (I thought rubocop would complain about this).
I think this format would be more preferred:

```suggestion
          payment_method_tags: payment_method_tags_by_id,
        }
```
Note that I've added an (unnecessary) comma at the end of the line. This practice makes for cleaner commit history, i.e. if we later add a new item to the hash, only the new line will show up in the diff.

Reply:
Ahh, I always wondered why the superfluous comma was added. Now it makes sense.

Reply:
Oops, I got mixed up. This is actually a list of parameters passed to a method, inside a block. So the extra comma wasn't valid. I've updated this again..